lib/src/models/song_recognition/song_recognition_options.dart

```diff
@@ -0,0 +1,22 @@
+class SongRecognitionOptions {
```

Comment:
Let's add our copyright signature here!
```
// Copyright (c) 2022, Jorge Rincón Arias
//
// Use of this source code is governed by an MIT-style
// license that can be found in the LICENSE file or at
// https://opensource.org/licenses/MIT.
```

Reply:
Only here? We should have like a _company_ copyright!

Reply:
Yeah, the issue is we still don't have a name. Maybe we could come up with one by the end of this week?